mysite/static/css/search/search.css

```diff
@@ -58,14 +58,15 @@ body#search #textual_search input[type='submit'] { width: 20%; padding: 5px; }
  *---------------------- */
 body#search .results-widget { color: #555; padding: 12px; padding-bottom: 6px; float: left; width: 97%; font-size: .75em; margin-bottom: 15px; border: 0px dashed #dedede; border-bottom-width: 1px; }
     body#search .results-widget:last-child { border-bottom-width: 0; border-top-width: 1px; }
-    body#search .results-widget > span { float: left; } 
-    body#search #results-summary { width: 50%; margin-right: 2%; } 
```

Comment:
Appreciate you getting rid of this stray whitespace, but it might have been better to do it in a separate commit to make this a little easier to review :)